tvb_build/hbp_openshift_deployer/tvb_backup.yml

```diff
@@ -0,0 +1,84 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: thevirtualbrain
+  annotations:
+    description: "Template to create a cronjob that copies the database data into a backup volume"
+    tags: "tvb, thevirtualbrain, hbp"
+parameters:
+  - name: POSTGRESQL_USER
+    displayName: PostgreSQL default user
+    description: User which will be used as default one for the PostgreSQL instance
+    value: "tvb_db_user"
+    required: true
+
+  - name: POSTGRESQL_PASSWORD
+    description: PostgreSQL DB password. This value will be autogenerated
+    generate: expression
+    from: "[a-zA-Z0-9]{15}"
+
+  - name: POSTGRESQL_DATABASE
+    displayName: PostgreSQL database name
+    description: PostgreSQL database name
+    value: "tvb_hbp_database"
+    required: true
+
+objects:
+  - kind: "PersistentVolumeClaim"
+    apiVersion: v1
+    metadata:
+      name: backup
+      annotations:
+        volume.beta.kubernetes.io/storage-class: managed-nfs-storage
+        volume.beta.kubernetes.io/storage-provisioner: fuseim.pri/ifs
+    spec:
+      accessModes:
+        - "ReadWriteOnce"
+      resources:
+        requests:
+          storage: "2Ti"
+
+  - kind: CronJob
+    apiVersion: batch/v1beta1
+    metadata:
+      name: tvb-postgres-backup
+    spec:
+      # ┌───────────── minute (0 - 59)
+      # │ ┌───────────── hour (0 - 23)
+      # │ │ ┌───────────── day of the month (1 - 31)
+      # │ │ │ ┌───────────── month (1 - 12)
+      # │ │ │ │ ┌───────────── day of the week (0 - 6) (Sunday to Saturday;
+      # │ │ │ │ │                                   7 is also Sunday on some systems)
+      # │ │ │ │ │
+      # │ │ │ │ │
+      # * * * * *
+      schedule: "0 12 * * 6"
+      concurrencyPolicy: "Forbid"
+      suspend: false
+      successfulJobsHistoryLimit: 7
+      failedJobsHistoryLimit: 7
+      jobTemplate:
+        spec:
+          template:
+            spec:
+              containers:
+                - name: tvb-postgres-backup
+                  image: docker-registry.ebrains.eu/okd_tools/postgresql_backups:1.1
```

Comment:
Here we should change after we have a new release with ubuntu:latest and a newer pg_dump version